llm_evals_w_crowdlab/llm_evals_w_crowdlab.ipynb

```diff
@@ -0,0 +1,7080 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "847d458d",
+      "metadata": {},
+      "source": [
+        "# LLM Evals with humans, AI judges, and GPT token probabilities."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "view-in-github"
+      },
+      "source": [
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/cleanlab/examples/blob/master/llm_evals_w_crowdlab/llm_evals_w_crowdlab.ipynb)     "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "dc2395c9-877b-44cc-97aa-715f133ab5fb",
+      "metadata": {
+        "id": "dc2395c9-877b-44cc-97aa-715f133ab5fb"
+      },
+      "source": [
```

Comment:
It seems like you don't plan to have any text re what this example is about, so please link to the blogpost here at the top.

Remember our users browse the tutorials & examples, these aren't just things linked from blogposts only.  We use cleanlab-tools repo instead for code that is solely shared for reproducibility/details to accompany content.

Reply:
Added!